android/templates/module/generated/build.gradle

```diff
@@ -120,8 +122,8 @@ android {
 	}
 	// TODO: Add Java 8 support in the future.
 	compileOptions {
-		sourceCompatibility JavaVersion.VERSION_1_7
-		targetCompatibility JavaVersion.VERSION_1_7
+		sourceCompatibility JavaVersion.VERSION_1_8
+		targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Can you remove the `// TODO:` comment please?

Reply:
Do we need to do the below so that Kotlin can support Java 8?
Google's docs suggest that we do. Would you mind confirming please?
   https://developer.android.com/studio/write/java8-support

```groovy
android {
	kotlinOptions {
		jvmTarget = '1.8'
	}
}
```

Reply:
I have totally missed to add that. I ran a double check and yes - it is required if we want to use Java 8 features in Kotlin files. I did a test with defining a static method with a body in an interface class and access it from a native module. Without this option it resulted in the following exception:
`Kotlin: Calls to static methods in Java interfaces are prohibited in JVM target 1.6. Recompile with '-jvm-target 1.8'` 
After adding them the module compiled fine and the method call was successful.